include/crypto/x509.h

```diff
@@ -62,7 +68,7 @@ struct x509_sig_info_st {
 
 struct X509_req_info_st {
     ASN1_ENCODING enc;          /* cached encoding of signed part */
-    ASN1_INTEGER *version;      /* version, defaults to v1(0) so can be NULL */
+    int ASN1_INTEGER = 0;      /* CSR v1 will always be used */
```

Comment:
This is not valid C.

`ASN1_INTEGER` is a type, not the name of a structure member, and you can't give default values like this. In addition, changing from a pointer to an integer type would be a change of API (even if this was valid C).